src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -38,17 +28,22 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         /// <summary>
         /// Invoked before redirecting to the identity provider to authenticate.
         /// </summary>
-        Task RedirectToAuthenticationEndpoint(RedirectContext context);
+        Task RedirectToIdentityProvider(RedirectContext context);
```

Comment:
I agree it does not sound awesome, but it has the advantage of being a clear correspective of the events that have been used for handling the same phase in the last 2 generations of auth software. Unless we have a new name that has excellent explicative power and does not require deep protocol knowledge, this looks the alternative with the least impact.